eta/modules/sample_videos.py

```diff
@@ -182,6 +197,27 @@ def _process_video(input_path, output_frames_dir, parameters):
         input_path, frames, output_patt=output_patt, size=size, fast=True)
 
 
+def _check_input_video_size(
+        stream_info, max_video_file_size, max_video_duration):
+    if max_video_file_size is not None:
+        file_size = int(stream_info.format_info["size"])
```

Comment:
you should use `stream_info.size_bytes` for this. Use properties whenever possible